samples/Blazor/BlazorWasm/BlazorWasm.Server/Api/TodoController.cs

```diff
@@ -1,95 +1,89 @@
-﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc;
 using Orleans;
 using BlazorWasm.Grains;
 using BlazorWasm.Models;
-using System;
 using System.Buffers;
 using System.Collections.Immutable;
 using System.ComponentModel.DataAnnotations;
-using System.Threading.Tasks;
 
-namespace Sample.Silo.Api
+namespace Sample.Silo.Api;
+
+[ApiController]
+[Route("api/todo")]
+public class TodoController : ControllerBase
 {
-    [ApiController]
-    [Route("api/todo")]
-    public class TodoController : ControllerBase
-    {
-        private readonly IGrainFactory factory;
+    private readonly IGrainFactory _factory;
 
-        public TodoController(IGrainFactory factory)
-        {
-            this.factory = factory;
-        }
+    public TodoController(IGrainFactory factory) => _factory = factory;
 
-        [HttpGet("{itemKey}")]
-        public Task<TodoItem> GetAsync([Required] Guid itemKey) =>
-            factory.GetGrain<ITodoGrain>(itemKey).GetAsync();
+    [HttpGet("{itemKey}")]
+    public Task<TodoItem?> GetAsync([Required] Guid itemKey) =>
+        _factory.GetGrain<ITodoGrain>(itemKey).GetAsync();
 
-        [HttpDelete("{itemKey}")]
-        public async Task DeleteAsync([Required] Guid itemKey)
-        {
-            await factory.GetGrain<ITodoGrain>(itemKey).ClearAsync();
-        }
+    [HttpDelete("{itemKey}")]
+    public Task DeleteAsync([Required] Guid itemKey) =>
+        _factory.GetGrain<ITodoGrain>(itemKey).ClearAsync();
 
-        [HttpGet("list/{ownerKey}", Name = "list")]
-        public async Task<ImmutableArray<TodoItem>> ListAsync([Required] Guid ownerKey)
-        {
-            // get all item keys for this owner
-            var keys = await factory.GetGrain<ITodoManagerGrain>(ownerKey).GetAllAsync();
+    [HttpGet("list/{ownerKey}", Name = "list")]
+    public async Task<ImmutableArray<TodoItem>> ListAsync([Required] Guid ownerKey)
+    {
+        // get all item keys for this owner
+        var keys = await _factory.GetGrain<ITodoManagerGrain>(ownerKey).GetAllAsync();
 
-            // fast path for empty owner
-            if (keys.Length == 0) return ImmutableArray<TodoItem>.Empty;
+        // fast path for empty owner
+        if (keys.Length is 0) return ImmutableArray<TodoItem>.Empty;
 
-            // fan out and get all individual items in parallel
-            var tasks = ArrayPool<Task<TodoItem>>.Shared.Rent(keys.Length);
-            try
+        // fan out and get all individual items in parallel
+        var tasks = ArrayPool<Task<TodoItem?>>.Shared.Rent(keys.Length);
```

Comment:
This pre-existing code is overkill, we should use the most obvious code here (new list, not a rented array).

Someone brought this up in Discord, so we have an anecdatum that it can cause confusion

Reply:
I'll fix this up and I agree, thanks @ReubenBond